sapi_manifests/madtom_lab/template

```diff
@@ -66,42 +65,13 @@
         { "processType": "agent", "port": 9080 }
```

Comment:
"agent" entries could go as well. These are the "marlin-agent" on CNs (https://github.com/joyent/manta-marlin/blob/master/agent/etc/agentconfig.coal.json#L4).

Reply:
Yes, dropped.